ecosystem/typescript/sdk/src/aptos_types/transaction.ts

```diff
@@ -474,6 +514,22 @@ export class TransactionArgumentU128 extends TransactionArgument {
   }
 }
 
+export class TransactionArgumentU256 extends TransactionArgument {
+  constructor(public readonly value: Uint256) {
+    super();
+  }
+
+  serialize(serializer: Serializer): void {
+    serializer.serializeU32AsUleb128(2);
```

Comment:
Why do we do u128 first? Does this not truncate?

Reply:
Same as above, but I probably messed up the tags... Will get that fixed.